### PR TITLE
Updates imagenet,mnist examples with result tables

### PR DESCRIPTION
**Pull request by andsteing**
As part of #707

Examples were run with code from #694

### CODE REVIEW

**Comment by myagues** [comment]
There is a typo in [`linen_examples/imagenet/train.py#L286`](https://github.com/google/flax/blob/9a0b06d24add079c9605e4ccb3cd36a370b56773/linen_examples/imagenet/train.py#L286), when defining `steps_per_eval`, that causes the model to iterate over validation data multiple times. This may explain the increased training time compared to [`examples/imagenet/imagenet_lib.py#L264`](https://github.com/google/flax/blob/9a0b06d24add079c9605e4ccb3cd36a370b56773/examples/imagenet/imagenet_lib.py#L264), where `steps_per_eval` is correctly set.

File: linen_examples/imagenet/README.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ This example uses linear learning rate warmup and cosine learning rate schedule.
...
+| Name                    |   Steps | Walltime   | Top-1 accuracy   | Metrics                                                                                                                               | Workdir                                                                                                                                                              |
+|:------------------------|--------:|:-----------|:-----------------|:--------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| v100_x8                 |  250199 | 18.0h      | 76.62%           | [tfhub.dev](https://tensorboard.dev/experiment/6yEYynrdRS2HZqHAD2ejPQ/#scalars&_smoothingWeight=0&regexInput=v100_x8)                 | [gs://flax_public/examples/imagenet/v100_x8](https://console.cloud.google.com/storage/browser/flax_public/examples/imagenet/v100_x8)                                 |
```
Comment (avital) [inline linen_examples/imagenet/README.md:15]: It really is concerning that this got so much slower. Do we have any lead on why?

**Review by avital — APPROVED** [review]
Please see my comment on the ImageNet training getting slower. Someone should investigate this. Do you know if it's happening? Or should we find someone to own this investigation

File: linen_examples/imagenet/README.md (hunk 1 above)
Comment (andsteing) [reply linen_examples/imagenet/README.md:15]: Rerunning it with #717 as suggested by @myagues above... Will update here when the results are ready.

Comment (jheek) [reply linen_examples/imagenet/README.md:15]: A this explains why I could not find anything during profiling. The steps are fast there’s just too many

Comment (andsteing) [reply linen_examples/imagenet/README.md]: Ok that fixed it :) We're now even faster & more accurate (at least in this run ;-)

**Comment by andsteing** [comment]
> Oh! So do we match the pre-Linen walltime now?

We surpassed it !

**Pull request merged**